rebasehelper/application.py

```diff
@@ -351,7 +351,7 @@ def patch_sources(self, sources):
         git_helper = GitHelper(sources[0])
         if not self.conf.non_interactive:
             git_helper.check_git_config()
-        patch = Patcher(self.conf.patchtool)
+        patch = Patcher("git")
```

Comment:
Ah OK, I thought, one day, you can want to add new patcher tool, so I left it there as is, only removed the option. I can cleanup the code in next iteration.